### ISSUE TITLE
How to identify how many handler are used in logger

### ISSUE THREAD

**Issue by Lazloo** [Delgan/loguru#201]
Hey,

I would like to identify the number of handlers that are associated with my logger object. Until now I used logger._handlers. But apparently, that does not exists anymore in version 0.4.

I used _handlers as follows 

```
from loguru import logger

def init_loguru(self, func_name=''):

    if len(logger._handlers) <= 1:
        logger.add(sink='log.txt', level='INFO')
        
    return logger
```

This function makes sure that the logger object contains at maximum to handlers.

I hope you can help
Lazloo

**Comment by Delgan** [comment]
Hi @Lazloo.

There is no method to know the number of existing handlers because I haven't found any use case for it. Handlers should preferably be added in one place, so you should be able to know how many there are.
If your program is used as a script, you are responsible for managing the handlers. If your program is used as lib, you should not add any handlers.

The `_handler` attribute is not part of the public API. Its equivalent in v0.4.0 is `logger._core.handlers`. You can use it, but I can't guarantee it exists forever... :wink:

**Comment by Lazloo** [comment]
Hey @Delgan ,

For me, the functionality to check if I already created a handler is very useful. In particular, when I use loguru inside of a class. Each object shall log into the same log-file. Without checking if a handler already exists, I would multiply the log message after each initialization of an object. I hope you will not remove this info.  ^^

THX 
Lazloo

**Comment by Delgan** [comment]
I think you should use a variable or method to make sure handlers are added only once. Surely you can also move the `.add()` call outside of your classes.

It is difficult for me to find an alternative solution without a concrete example. I think it's user responsibility to manage handlers configuration, that's why I don't plan to add such feature. However, if I ever change the private API in the future, I'll make sure to let you know. :wink:
